public/app/features/plugins/datasource_srv.ts

```diff
@@ -148,7 +155,7 @@ export class DatasourceSrv implements DataSourceService {
     return metricSources;
   }
 
-  addDataSourceVariables(list) {
+  addDataSourceVariables(list: any) {
```

Comment:
we have a Variable interface defined in app/types/templates.ts

Reply:
This also uses the properties `value, meta, sort` that are not on the Variable interface. Should I add those to the interface?